webapp/app/js/perf.js

```diff
@@ -203,35 +217,57 @@ perf.factory('PhCompare', [ '$q', '$http', 'thServiceDomain', 'PhSeries', 'math'
       return cmap;
     },
 
-    getResultsMap: function(projectName, seriesList, timeRange, resultSetId) {
+    getInterval: function(oldTimestamp, newTimestamp) {
+      var now = (new Date()).getTime() / 1000;
+      var timeRange = Math.min(oldTimestamp, newTimestamp);
+      timeRange = Math.round(now - timeRange);
+
+      //now figure out which predefined set of data we can query from
+      var intervals = [86400, 604800, 1209600, 2592000, 5184000, 7776000];
```

Comment:
So this is already used here: https://github.com/mozilla/treeherder-ui/blob/master/webapp/app/js/graphs.js#L16

We should create an angular value for this so we can reuse this information here, there, and elsewhere. The keys should be the human readable name, the value the integer values we send to treeherder/perfherder (for cases we just need the values, just get the values). Some examples here: https://github.com/jmaher/treeherder-ui/blob/subtest/webapp/app/js/values.js and